### PR TITLE
setup babel with node

### PR DESCRIPTION
**Pull request by haegemonia76**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "license": "OSL-3.0",
   "engines": {
-    "node": ">=12 <13"
+    "node": "12"
```
Comment (devaquila) [inline package.json:8]: "node": ">=12 <13"

File: jsconfig.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "baseUrl": ".",
+        "paths": {
+            "Utils/*": ["./utils/*"],
+            "Services/*": ["./services/*"],
+            "Routes/*": ["./routes/*"],
+            "Config/*": ["./config/*"],
+            "Orm/*": ["./orm/*"],
```
Comment (devaquila) [inline jsconfig.json:9]: Unexpected character ('}' (code 125)): was expecting double-quote to start field name

File: package.json (hunk 1 above)
Comment (haegemonia76) [reply package.json:8]: it's the same thing, `>=12 <13` equals to `12`, it's easier to read

**Comment by devaquila** [comment]
Tips :
Setting in launch.json :
"runtimeExecutable":"babel-node"
babel-node need to be install in global and settings PATH

**Comment by devaquila** [comment]
Change to nodejs14

**Pull request closed without merging**